### PR TITLE
Update big data failed cases due to changes of PR #170

### PR DESCRIPTION
**Pull request by fenglixa**
**Which issue is resolved by this Pull Request:** 
After PR #170 and PR #169 merged, the case of `test_bigger_data_workflow` will be failed as below, because of incompatibilities:

```
======================================================================
FAIL: test_bigger_data_workflow (compiler.compiler_tests.TestTektonCompiler)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/fengli/go/src/github.com/kubeflow/kfp-tekton/sdk/python/tests/compiler/compiler_tests.py", line 249, in test_bigger_data_workflow
    self._test_pipeline_workflow(file_passing_pipelines, 'big_data_passing.yaml', generate_pipelinerun=True)
  File "/Users/fengli/go/src/github.com/kubeflow/kfp-tekton/sdk/python/tests/compiler/compiler_tests.py", line 304, in _test_pipeline_workflow
    self._verify_compiled_workflow(golden_yaml_file, compiled)
  File "/Users/fengli/go/src/github.com/kubeflow/kfp-tekton/sdk/python/tests/compiler/compiler_tests.py", line 367, in _verify_compiled_workflow
    self.assertEqual(golden, compiled_workflow)
AssertionError: Lists differ: [{'ap[20009 chars]': {'name': 'file-passing-pipelines-run'}, 'sp[403 chars]m'}}] != [{'ap[20009 chars]': {'annotation': {'tekton.dev/input_artifacts[492 chars]m'}}]
... ...
   {'apiVersion': 'tekton.dev/v1beta1',
    'kind': 'PipelineRun',
+   'metadata': {'annotation': {'tekton.dev/input_artifacts': '{}',
+                               'tekton.dev/output_artifacts': '{}'},
-   'metadata': {'name': 'file-passing-pipelines-run'},
?   ^^^^^^^^^^^^^

+                'name': 'file-passing-pipelines-run'},
?   ^^^^^^^^^^^^^

```

**Description of your changes:**
Fix it by adding annotation to the yaml files of big data.

### CODE REVIEW

**Comment by ckadner** [comment]
@fenglixa -- can you do a sweep and replace all occurrences of the term `bigger_data_passing` with `big_data_passing` including files/folder names?

**Comment by fenglixa** [comment]
> @fenglixa -- can you do a sweep and replace all occurrences of the term `bigger_data_passing` with `big_data_passing` including files/folder names?

OK, let me do it in this PR.

**Comment by fenglixa** [comment]
> @fenglixa -- can you do a sweep and replace all occurrences of the term `bigger_data_passing` with `big_data_passing` including files/folder names?

Updated.
/cc @ckadner @Tomcli , could you please help review? Thanks.

File: samples/big_data_passing/big_data_passing_description.ipynb [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "\n",
     "Some examples of typical types of small data are: number, URL, small string (e.g. column name).\n",
     "\n",
-    "Small lists, dictionaries and JSON structures are fine, but keep an eye on the size and consider switching to file-based data passing methods taht are more suitable for bigger data (more than several kilobytes) or binary data.\n",
+    "Small lists, dictionaries and JSON structures are fine, but keep an eye on the size and consider switching to file-based data passing methods taht are more suitable for big data (more than several kilobytes) or binary data.\n",
```
Comment (animeshsingh) [inline samples/big_data_passing/big_data_passing_description.ipynb:46]:
```suggestion
    "Small lists, dictionaries and JSON structures are fine, but keep an eye on the size and consider switching to file-based data passing methods that are more suitable for big data (more than several kilobytes) or binary data.\n",
```

File: samples/big_data_passing/big_data_passing_description.ipynb [hunk 2]
```diff
@@ -504,7 +504,7 @@
     "\n",
     "Note on input/output names: When the function is converted to component, the input and output names generally follow the parameter names, but the \"\\_path\" and \"\\_file\" suffixes are stripped from file/path inputs and outputs. E.g. the `number_file_path: InputPath(int)` parameter becomes the `number: int` input. This makes the argument passing look more natural: `number=42` instead of `number_file_path=42`.\n",
     "\n",
-    "Notes: As we used 'workspaces' in tekton pipelines to handle the bigger data processing, the complier will generator the PVC definations,it needs the volume to store the data.\n",
+    "Notes: As we used 'workspaces' in tekton pipelines to handle the big data processing, the complier will generator the PVC definations,it needs the volume to store the data.\n",
```
Comment (animeshsingh) [inline samples/big_data_passing/big_data_passing_description.ipynb:507]:
```suggestion
    "Notes: As we used 'workspaces' in Tekton pipelines to handle big data processing, the compiler will generate the PVC definitions and needs the volume to store the data.\n",
```

File: sdk/python/kfp_tekton/compiler/_data_passing_rewriter.py [hunk 3]
```diff
@@ -61,9 +61,9 @@ def fix_big_data_passing(
     3. Propagate the consumption information upstream to all inputs/outputs all
        the way up to the data producers.
     4. Convert the inputs, outputs based on how they're consumed downstream.
-    5. Use workspaces instead of result and params for bigger data passing.
-    6. Added workspaces to tasks, pipelines, pipelineruns, if the parmas is bigger data.
-    7. A PVC named with pipelinerun name will be created if bigger data passing, as workspaces need to use it.
+    5. Use workspaces instead of result and params for big data passing.
+    6. Added workspaces to tasks, pipelines, pipelineruns, if the parmas is big data.
+    7. A PVC named with pipelinerun name will be created if big data passing, as workspaces need to use it.
```
Comment (animeshsingh) [inline sdk/python/kfp_tekton/compiler/_data_passing_rewriter.py:66]:
```suggestion
    7. A PVC named with pipelinerun name will be created if big data is passed, as workspaces need to use it.
```

File: sdk/python/kfp_tekton/compiler/_data_passing_rewriter.py [hunk 4]
```diff
@@ -608,7 +608,7 @@ def big_data_passing_tasks(task: dict, inputs_tasks: set,
     return task
 
 
-# Create pvc for pipelinerun if bigger data passing.
+# Create pvc for pipelinerun if big data passing.
```
Comment (animeshsingh) [inline sdk/python/kfp_tekton/compiler/_data_passing_rewriter.py:611]:
```suggestion
# Create pvc for pipelinerun if using big data passing.
```

**Comment by fenglixa** [comment]
Thanks @animeshsingh , I addressed your comments, please help check whether it's OK for you now?

**Pull request merged**